ui/index.d.ts

```diff
@@ -178,19 +178,25 @@ export declare abstract class AzExtTreeItem {
     /**
      * This is is used for the AzureTreeItem.openInPortal action. It is also used per the following documentation copied from VS Code:
      * Optional id for the treeItem that has to be unique across tree. The id is used to preserve the selection and expansion state of the treeItem.
-     *
-     * If not provided, an id is generated using the treeItem's label. **Note** that when labels change, ids will change and that selection and expansion state cannot be kept stable anymore.
```

Comment:
It makes sense to get rid of this comment then, right?  Since the label is being added only in `fullId`

Reply:
No we should keep this line. First of all it's copied straight from VS Code docs (as line 179 states): https://github.com/microsoft/vscode/blob/main/src/vs/vscode.d.ts#L9307

Secondly, they never say this specific property will return the label. They say "_an_ id is generated", meaning some under-the-covers id will use the label. Also, they say "if not provided", meaning this property can return undefined.